### DISCUSSION TITLE
Ideas for a better leveling experience

### DISCUSSION THREAD

**Discussion by elitechobohax** [chromiecraft/chromiecraft#4018] [Suggestions]
We've all leveled countless of characters to max level countless of times on countless of servers, from retail to privates. What i am asking for is a better leveling experience, that is a little less time consuming - Manye people have gotten jobs, lives and wifes and all that over the years.

Experience Rates: Please, for the love of all that is holy, increase this to x3. I do not want to play on Warmane and their Pay2Win crap, but it is honestly preferable to doing a very boring x1 grind or we could play Retail classic.

Abilities: Please enable auto learning of skills or if you want to see people pay for them, some kind of gimmick so that they can buy the skills while out adventuring (Think Book of Ascension from the Ascension server)

Dual Spec: This should be enabled from level 10 with a cost of 1 or 10gold. All Dual spec does, is make grouping considerably more viable and better.

Healer "boon": Let whoever is queued up for a dungeon and designated as the healer, be able to roll need on any type of armor, until lets say, level 55+ dungeons. Paladins, Shamans and Druids are all excellent healers, however, their caster gear choices are severely limited with using leather/mail/plate and the majority of the gear they want to use, will be cloth.

**Comment by Amrx2** [comment]
Hi there, we already offer double xp weekends every weekend, boosted exp gain in BGs as well as a Recruit a Friend system giving you permanent rested EXP until 55 or 30 days (whatever is hit first). Also being wrath based means your journey to 60 is a lot faster than retail classic. 

Abilities need to be learnt from trainers or their respective quests, it's important for us people can play from an original wrath client without custom patches so you won't see stuff like auto trained skills here or custom shop books to train skills wherever. 

Dual spec is for the discounted price of 800g currently and will go up to 1000g when Outlands opens. It's from level 40 learnable. 

That's already the case here for loot in those dungeons as well for classes like holy paladins being able to need on cloth, if I remember up to 55 dungeons then normal loot rules apply. 

Thanks for your suggestions but these do appear more for a custom fun server or one with a shorter life span. We have been up for just shy of 2 years soon and aren't even 1/3 through out planned content, you will have plenty of time to do everything you want to here on Chromiecraft! 

Cheers,
Construct

**Reply by elitechobohax** [reply]
Thank you for the response. Not what i was looking for then.

**Reply by Amrx2** [reply]
No worries, fingers crossed you find the server you are looking for going forward. Thanks for taking the time to consider Chromiecraft however and good luck!
